### PR TITLE
Project index

### PR DESCRIPTION
**Pull request by malafalabella**
### Added page titles, technology section, navbar, footer and CSS transitions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
```
Comment (arconejeros) [inline index.html]: add a `/` to close all of this self-closing tags like ` <meta charset="utf-8" /> `

File: index.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+
+  <!-- HEADER SECTION -->
```
Comment (arconejeros) [inline index.html]: remove all of those empty lines and comments

File: index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  <div class="navbar">
+    <img class="item-1" src="./assets/wolox_logo.svg"/>
```
Comment (arconejeros) [inline index.html]: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  add that prop to all the images

File: index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+  <!-- FOOTER SECTION -->
+  <div class="footer">
+    <div class="item-1">WE ARE GAME CHANGERS</div>
+    <img src="./assets/wolox_logo_white.svg"/>
+    <div class="item-2">Copyright 2018. Wolox. All rights reserved</div>
+  </div>
+
+</body>
+</html>
```
Comment (arconejeros) [inline index.html]: an empty line should be at the bottom of every file. add it here

File: scss/_constant.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+$white : #ffffff;
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
```
Comment (arconejeros) [inline scss/_constant.scss]: add a empty line here

File: scss/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
+.footer{
```
Comment (arconejeros) [inline scss/_footer.scss]: add a empty space between class name and bracket in every class statement

File: scss/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    color: $white;
+    align-self: center;
+  }
+
+  .item-2{
+    color: $white;
+    align-self: flex-end;
+    padding: 0 10px 0 0;
+  }
+}
```
Comment (arconejeros) [inline scss/_footer.scss]: add a empty line here

File: scss/_navbar.scss [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    width: 200px;
+    padding: 20px;
+  }
+
+  .item-2{
+    align-self: center;
+    padding: 0 20px 0 0;
+    letter-spacing: 2px;
+  }
+
```
Comment (arconejeros) [inline scss/_navbar.scss]: remove tis line

File: scss/_navbar.scss [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+}
```
Comment (arconejeros) [inline scss/_navbar.scss]: add a empty line here

File: scss/_reset.scss [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+body,
+ul,
+ol {
+  margin:  0;
+  padding: 0;
+}
+
+body{
+  font-family: 'Roboto', sans-serif;
+}
```
Comment (arconejeros) [inline scss/_reset.scss]: add a empty line here

File: scss/_tech.scss [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+          .tech-description{
+            margin: 0px 20px;
+            text-align: center; 
+            flex-basis: 30%;
+          }
+
+        }
+    }
+  }
+}
```
Comment (arconejeros) [inline scss/_tech.scss]: add a empty line here

File: scss/_titles.scss [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+  
+  img {
+    height: 200px;
+    width: 200px;  
+  }
+
+  span {
+    font-size: $text-big;
+    align-self: center;
+    text-indent: -55px;
```
Comment (arconejeros) [inline scss/_titles.scss]: avoid using negative values

File: scss/_titles.scss [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+  }
+}
```
Comment (arconejeros) [inline scss/_titles.scss]: add a empty line here

File: scss/application.scss [hunk 14]
```diff
@@ -0,0 +1,6 @@
+@import '_reset';
+@import '_constant';
+@import '_titles';
+@import '_tech.scss';
+@import '_navbar';
+@import '_footer';
```
Comment (arconejeros) [inline scss/application.scss]: add a empty line here

File: index.html [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+</head>
+<body>
+
+  <!-- HEADER SECTION -->
+  <div class="navbar">
+    <img class="item-1" src="./assets/wolox_logo.svg"/>
+    <span class="item-2">Developer Tools</span>
+  </div>
+
+  <!-- FRONTEND TOOLS SECTION -->
```
Comment (arconejeros) [inline index.html]: same as above about emty line and comments

File: index.html [hunk 16]
```diff
@@ -0,0 +1,144 @@
...
+                <img src="./assets/polymer_icon.svg" />
+                <div class="tech-heading">Polymer</div>
+                <div class="tech-release">Release: 2015</div>
+                <div class="tech-description">Polymer is an open-source JavaScript library for building web applications using Web Components.</div>
+            </div>
+        </div>
+    </div>
+  </div>
+
+  <!-- BACKEND TOOLS SECTION -->
```
Comment (arconejeros) [inline index.html]: same as above

File: index.html [hunk 17]
```diff
@@ -0,0 +1,144 @@
...
+                <div class="tech-heading">Django</div>
+                <div class="tech-release">Release: 2005</div>
+                <div class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh) is a free and open-source web framework written in Python, which follows the model-view-template(MVT) architectural pattern.</div>
+            </div>
+        </div>
+      </div>
+
+  </div>
+
+  <!-- FOOTER SECTION -->
```
Comment (arconejeros) [inline index.html]: same as above

File: index.html (hunk 1 above)
Comment (malafalabella) [reply index.html]: pushed the changes

File: index.html (hunk 2 above)
Comment (malafalabella) [reply index.html]: pushed the changes

File: index.html (hunk 3 above)
Comment (malafalabella) [reply index.html]: pushed the changes

File: index.html (hunk 4 above)
Comment (malafalabella) [reply index.html]: pushed the changes

File: scss/_constant.scss (hunk 5 above)
Comment (malafalabella) [reply scss/_constant.scss]: pushed the changes

**Pull request merged**